augur/io/sequences.py

```diff
@@ -19,7 +24,7 @@ def read_sequence(
 
 def read_sequences(
     *paths: Iterable[str],
-    format: str = "fasta",
+    format: str = BIOPYTHON_FASTA_FORMAT,
```

Comment:
I don't quite understand why we don't do the BIOPYTHON_FASTA_FORMAT version switch for biopython version _inside_ this function.

Rather than requiring users (even internal ones) to pass the fasta argument, we should just rewrite `fasta` -> `fasta-pearson` inside the function here and note this in the docstring and our changelog.

Reply:
Done:

https://github.com/nextstrain/augur/blob/5c5325605f575ed86fd36211758573991450d986/augur/io/sequences.py#L20-L22